src/Stormpath/Http/HttpClientRequestExecutor.php

```diff
@@ -29,10 +30,10 @@ class HttpClientRequestExecutor implements RequestExecutor
     private $httpClient;
     private $signer;
 
-    public function __construct()
+    public function __construct(RequestSigner $signer)
     {
         $this->httpClient = new Client();
```

Comment:
However, I would set the default value of the argument to null and, if it is null, create an `Sauthc1Signer`.